pkgs/development/libraries/fltk/default.nix

```diff
@@ -39,10 +54,18 @@ composableDerivation.composableDerivation {} rec {
     localpngSupport = false;
     sharedSupport = true;
     threadsSupport = true;
-  };
+  } // cfg;
 
   meta = {
-    description = "a C++ cross platform lightweight gui library binding";
+    description = "A C++ cross-platform lightweight GUI library";
     homepage = http://www.fltk.org;
+    platforms = stdenv.lib.platforms.linux;
+    license = stdenv.lib.licenses.gpl2;
   };
+
+  # doesn't build without it. Why?
+  patches = [
```

Comment:
That's why :) https://github.com/NixOS/nixpkgs/pull/15622/files#diff-16b346e3b0c0f7128a8404bb4fb4a07cR19
Just rename `postPatch` to something like `preConfigure` or `patchPhase`. If you don't have patches ofc.